src/snapshot/Cargo.toml

```
@@ -6,11 +6,9 @@ edition = "2018"
 autobenches = false
 
 [dependencies]
-serde = ">=1.0.27"
-serde_derive = ">=1.0.27"
 bincode = "1.2.1"
-versionize = { version = "0.1.1" }
-versionize_derive = { git = "https://github.com/firecracker-microvm/versionize_derive", tag = "v0.1.0" }
+versionize = { version = "0.1.2" }
+versionize_derive = { version = "0.1.1" }
```

Should we take out the exact version enforcement?
```suggestion
versionize = { version = ">=0.1.2" }
versionize_derive = { version = ">=0.1.1" }
```

---

Indeed, I have updated all paths to ">=". Take a look!